src/uwtools/drivers/forecast.py

```diff
@@ -231,14 +231,47 @@ def _define_boundary_files(self) -> Dict[str, str]:
 
         return boundary_files
 
+    def _prepare_and_run_batch_script(self, run_directory: str) -> bool:
+        """
+        Prepares and runs batch script.
+        """
+        batch_script = self.batch_script()
+        assert self._batch_script is not None
+        outpath = Path(run_directory) / self._batch_script
+
+        if self._dry_run:
+            # Apply switch to allow user to view the run command of config.
+            # This will not run the job.
+            log.info("Batch Script:")
+            batch_script.dump(None)
```

Comment:
Here, too, we should log the lines of the batch script instead of printing them to `stdout`. I also notice that we call the batch script `bs` elsewhere, and `batch_script` here, so those could be harmonized. Better yet: Can we factor this "log the lines of a batch script in dry-run mode" logic out? Then everything would be in sync.